app-config.yaml

```
@@ -15,9 +15,9 @@ backend:
   # Used for enabling authentication, secret is shared by all backend plugins
   # See https://backstage.io/docs/auth/service-to-service-auth for
   # information on the format
-  # auth:
-  #   keys:
-  #     - secret: ${BACKEND_SECRET}
+  auth:
+    keys:
+      - secret: ${BACKEND_SECRET}
```

```suggestion
      - secret: ${BACKEND_AUTH_SECRET}
```

The env var should be more specific.

---

Fixed